features/history.yml.dist

```diff
@@ -0,0 +1,73 @@
+# Generated from: history.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2020-01-15
+  baseline_high_date: 2022-07-15
+  support:
+    chrome: "46"
+    chrome_android: "46"
+    edge: "79"
+    firefox: "46"
+    firefox_android: "46"
+    safari: "11"
+    safari_ios: "11"
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "1"
+  #   safari_ios: "1"
+  - api.History
+  - api.History.back
+  - api.History.forward
+  - api.History.go
+  - api.History.length
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "4"
+  #   firefox_android: "4"
+  #   safari: "5"
+  #   safari_ios: "4"
+  - api.History.pushState
```

Comment:
This is the support data that matches caniuse.com. So, it seems like everything up to this point is really necessary to use the API, but the 2 following keys are nice-to-haves (`state` and `scrollRestoration`). Reading their documentation, it does seem like you can use the API without them.
So I would be tempted to use `compute_from` on `api.History.pushState` to match caniuse.com.